airbyte-webapp/src/views/Connection/ConnectionForm/components/SyncCatalogField.tsx

```diff
@@ -17,20 +17,15 @@ import { BulkHeader } from "../../CatalogTree/components/BulkHeader";
 import { ConnectionFormMode } from "../ConnectionForm";
 import InformationToolTip from "./InformationToolTip";
 import Search from "./Search";
+import styles from "./SyncCatalogField.module.scss";
 
 const TreeViewContainer = styled.div<{ mode?: ConnectionFormMode }>`
   margin-bottom: 29px;
   max-height: 600px;
   overflow-y: auto;
   --webkit-overlay: true;
-  // Find better way (for checkbox)
```

Comment:
Old way:
- The row div is only as wide as the shaded fields.  The checkbox is technically a part of the row but is thrown off to the side using these margins/width adjustments.  Flow of items was difficult to adjust this way.

New way:
- There is a row container containing the CheckboxCell, the ArrowCell, and then the catalog contents (the info about the stream)